### PR TITLE
Erlang OTP 19.1

### PR DESCRIPTION
**Pull request by aaronjensen**
https://github.com/erlang/otp/releases/tag/OTP-19.1

### CODE REVIEW

File: 19/slim/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 # We'll install the build dependencies, and purge them on the last step to make
 # sure our final image contains only what we've just built:
 RUN set -xe \
-	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="584e08c7d810204163247dd2187bfc44175bac60d3e2577a0ea55a6a898c8dce" \
-	&& runtimeDeps=' \
+  && OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
+  && OTP_DOWNLOAD_SHA256="caf320c07bdd4c6e11831a0b0d25645a29112007077dbf11eec22437f8b041ed" \
+  && runtimeDeps=' \
```
Comment (c0b) [inline 19/slim/Dockerfile]: this whole set looks clean, but could you not to change space format? it was a single tab

File: 19/Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-ENV OTP_VERSION="19.0.7"
+ENV OTP_VERSION="19.1"
 
 # We'll install the build dependencies for erlang-odbc along with the erlang
 # build process:
 RUN set -xe \
 	&& OTP_DOWNLOAD_URL="https://github.com/erlang/otp/archive/OTP-${OTP_VERSION}.tar.gz" \
-	&& OTP_DOWNLOAD_SHA256="584e08c7d810204163247dd2187bfc44175bac60d3e2577a0ea55a6a898c8dce" \
+  && OTP_DOWNLOAD_SHA256="caf320c07bdd4c6e11831a0b0d25645a29112007077dbf11eec22437f8b041ed" \
```
Comment (c0b) [inline 19/Dockerfile]: and here.

**Comment by aaronjensen** [comment]
Ugh, sloppy of me and my text editor, sorry. Fixed.

**Comment by aaronjensen** [comment]
@c0b anything else on this one?

**Comment by c0b** [comment]
just merged @aaronjensen would you mind also make a PR to official-images ?

https://github.com/docker-library/official-images/pulls?q=label:library/erlang

**Comment by aaronjensen** [comment]
@c0b https://github.com/docker-library/official-images/pull/2172

**Pull request merged**